grid/grid2.css

```diff
@@ -0,0 +1,40 @@
+*{
+    padding:0;
+    margin:0;
+    box-sizing: border-box;
+}
+.grid-container-sec{
+    display: grid;
+    grid-template-columns:1fr 1fr 1fr;
+    grid-template-rows: 200px;
+}
+:root{
+    --grid-item-one-color:aqua;
+    --gird-item-two-color:bisque;
+    --gird-item-three-color:salmon;
+}
+.item1{
+    width:15rem;
+    height: 12.5rem;
+    border:1px solid black;
+    background-color: var(--grid-item-one-color);
+    text-align: center;
+    padding:2rem;
+}
+.item2{
+    width:15rem;
+    height: 12.5rem;
+    border: 1px solid black;
+    background-color: var(--gird-item-two-color);
+    text-align: center;
+    padding:5rem;
+}
+.item3{
+    width:15rem;
+    height: 12.5rem;
+    border:1px solid black;
+    height: 12.5rem;
+    background-color:var(--gird-item-three-color);
+    text-align: center;
```

Comment:
Same issue as mentioned above. Try to create a common css class for common properties between item1 item2 and item3